styleguide/screens/Patterns/Cards/Cards.js

```diff
@@ -0,0 +1,653 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Specimen from '../../../components/Specimen/Specimen';
+import { D, H, T, C } from '../../../components/Scaffold/Scaffold';
+
+import Card from '../../../../components/Cards/Card/Card';
+import PictureCard from '../../../../components/Cards/PictureCard/PictureCard';
+import SpaceFeatureCard from '../../../../components/Cards/SpaceFeatureCard/SpaceFeatureCard';
+import EmptyListingCard from '../../../../components/Cards/EmptyListingCard/EmptyListingCard';
+import SpaceListingCard from '../../../../components/Cards/SpaceListingCard/SpaceListingCard';
+import Badge from '../../../../components/Badge/Badge';
+import CondensedSpaceCard from '../../../../components/Cards/CondensedSpaceCard/CondensedSpaceCard';
+import PlaceListingCard from '../../../../components/Cards/PlaceListingCard/PlaceListingCard';
+import EditorialCard from '../../../../components/Cards/EditorialCard/EditorialCard';
+import GuideCard from '../../../../components/Cards/EditorialCard/GuideCard';
+import EventCard from '../../../../components/Cards/EditorialCard/EventCard';
+
+import shared from '../../../../styleguide/shared.css';
+import m from '../../../../globals/modifiers.css';
+import css from './Cards.css';
+
+export default class CardDocumentation extends Component {
+  state = {
+    isSpaceFavourited: false,
+    isUserLoggedin: false,
+  };
+
+  handleFavouriteClick = () => {
+    this.setState(currentState => ({
+      ...currentState,
```

Comment:
You shouldn't need to do this? Anything that the `setState` fn returns is partial update, in the same way as passing a straight up object